core/src/scene/sceneLoader.cpp

```diff
@@ -744,12 +744,40 @@ void SceneLoader::parseStyleProps(Node styleProps, StyleParamMap& paramMap, cons
         } else if(propItr->second.IsMap()) {
             parseStyleProps(propItr->second, paramMap, paramKey);
         } else {
-            logMsg("Error: Badly formed Style property, need to be a scalar, sequence or map. %s will not be added to stype properties.\n", paramKey.c_str());
+            logMsg("Error: Badly formed Style property, need to be a scalar, sequence or map."
+                    "%s will not be added to stype properties.\n", paramKey.c_str());
         }
     }
 
 }
 
+void SceneLoader::loadSublayers(YAML::Node layer, std::vector<std::shared_ptr<SceneLayer>>& subLayers) {
+
+    for (auto subLayerItr = layer.begin(); subLayerItr != layer.end(); ++subLayerItr ) {
+
+        std::string subLayerName = subLayerItr->first.as<std::string>();
+        if( subLayerName != "data" && subLayerName != "filter" && subLayerName != "draw" &&
```

Comment:
Just got an idea for a syntax to separate keywords from sublayers: Keys don't have to be scalars so using a one item list like [sublayer]. This would allow to differentiate keyword from layers (and would also avoid some problems when introducing new keywords later on :)